releasenotes/notes/0.23/equivalence-to-graph-3b52912ecb542db8.yaml

```diff
@@ -2,12 +2,11 @@
 features:
   - |
     The :class:`~.EquivalenceLibrary` is now
-    represented internally as a ``PyDiGraph``. As part of this change,
-    there is a new public-facing attribute - ``graph``: the graph
-    representation of the EquivalenceLibrary. This attribute is
-    intended for use internally in qiskit and therefore should always
-    be copied before being modified by the user to prevent possible
-    corruption of the internal equivalence graph.
+    represented internally as a ``PyDiGraph``, this underlying graph object
```

Comment:
We intersphinx to rustworkx (actually retworkx, but the redirect works), so we could cross-ref this if we wanted.  Not important, though.

Reply:
I wasn't sure it was in the metapackage or not so I left this as is for now. We can always come back and update it later, it's not critical